packages/authentication/types/auth.ts

```diff
@@ -24,10 +25,30 @@ export interface LoginChangeExpiredPasswordRedirectResponse {
   redirectUrl: string
 }
 
+export interface AllAuthMethod {
```

Comment:
These types all already exist in `packages/authentication/types/allAuth.ts`. Use the existing types, or add what you need to there.